schema/a11y.schema.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://readium.org/webpub-manifest/schema/a11y.schema.json",
+  "title": "Accessibility Object",
+  "type": "object",
+  "properties": {
+    "conformsTo": {
+      "type": [
+        "string",
+        "array"
+      ],
+      "format": "uri",
+      "items": {
+        "type": "string",
+        "format": "uri"
+      }
+    },
+    "certification": {
+      "type": "object",
+      "properties": {
+        "certifier": {
+          "type": "string"
+        },
+        "credential": {
+          "type": "string"
+        },
+        "report": {
+          "type": [
+            "string",
+            "array"
```

Comment:
I don't see the logic in have several reports.

Reply:
This is something that we've inherited from the current W3C spec. We would have a hard time deciding which one to keep if we ever encountered multiple reports in an EPUB.

Reply:
You could argue in the same way about other properties.
As I understand it, EPUB allows several reports in order to have one report per certifier, thanks to `refine`. 
Therefore, I would allow several `certification`, each one having at most one `certifiedBy`, `credential` and `report`.